pkg/kubefed2/federate/federate.go

```diff
@@ -52,8 +53,8 @@ var (
 		flag otherwise.`
 
 	federate_example = `
-		# Federate resource named "my-dep" in namespace "my-ns" of type identified by FederatedTypeConfig "deployment.apps"
-		kubefed2 federate deployment.apps my-dep -n "my-ns" --host-cluster-context=cluster1`
+		# Federate resource named "my-dep" in namespace "my-ns" of kubernetes type "deploy"
+		kubefed2 federate deploy my-dep -n "my-ns" --host-cluster-context=cluster1`
```

Comment:
(No action required) Using the `deploy` short-name will result in an error message because it is ambiguous (it can mean `deployments.extensions` - the beta version - or `deployment.apps` - the GA version).  Will need to to fix before this PR series is done.